README.md

```diff
@@ -109,6 +109,10 @@ docker run --name openmrscd  -p 8080:8080 \
   mekomsolutions/openmrscd:latest
 ```
 
+### Using docker-compose
+
+```docker-compose up```
+
```

Comment:
@rbuisson I guess the landing README will have to be reviewed to put this in the right place?